subprojects/delivery/sign-service/src/main/kotlin/com/avito/android/signer/SignServicePlugin.kt

```diff
@@ -68,20 +62,4 @@ public class SignServicePlugin : Plugin<Project> {
             }
         }
     }
-
-    private fun checkSigningConfig(variant: ApplicationVariant) {
-        require(variant.signingConfig == null) {
```

Comment:
signingConfig can't be null anymore, despite comment in javadoc (it's a mistake)